differential_equations2.py

```diff
@@ -0,0 +1,40 @@
+#forward Euler method to numerically solve a logistic growth model
+#and then plots the solution using matplotlib
+from typing import Callable, List, Tuple
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def f(t: float, u: float) -> float:
+    alpha = 0.2
+    R = 1.0
+    unext = alpha*(u) * (1-(u/R))
+    return unext
+u0 = 0.1
+T = 40
+N = 10
+
+
+def forward_euler(f: Callable[[float, float], float], 
+    u0: float, 
+    T: float, 
+    N: int
+) -> Tuple[List[float], List[float]]:
+    """Solve u’=f(t, u), u(0)=u0, with n steps until t=T."""
```

Comment:
I would add an `Args:` block to this docstring describing the arguments (see [here](https://google.github.io/styleguide/pyguide.html#doc-function-args))